docs/api/api.md

```diff
@@ -0,0 +1,254 @@
+```
+
+#### POST request
+
+```bash
+curl --request POST \
+    --header "Content-Type: application/json" \
+    --data '{"name": "GhsPopComparisonBuildings", "layerName": "building_count", "dataset": "regions", "featureId": 3}' \
+    "https://oqt.ohsome.org/api/indicator"
+```
+
+> Tip: Use Python to pretty-print the response JSON -> `curl [...] | python -m json.tool`
+
+
+### Request an Indicator for custom bounding polygon(s)
+
+Since GeoJSON strings are quite big it is recommended that POST requests are used.
+
+#### POST request
+
+Let's write down the request parameters in a `JSON` file.
+Note that the bpolys parameters needs to be a valid GeoJSON object.
+
+```bash
+{
+  "name": "GhsPopComparisonBuildings",
+  "layerName": "building_count",
+  "bpolys": {
+    "type": "Polygon",
+    "coordinates": [
+      [
+        [
+          8.674092292785645,
+          49.40427147224242
+        ],
+        [
+          8.695850372314453,
+          49.40427147224242
+        ],
+        [
+          8.695850372314453,
+          49.415552187316095
+        ],
+        [
+          8.674092292785645,
+          49.415552187316095
+        ],
+        [
+          8.674092292785645,
+          49.40427147224242
+        ]
+      ]
+    ]
+  }
+}
+```
+
+Now that a `JSON` file exists on disk one can post it with `curl` using following syntax `curl -X POST -H "Content-Type: application/json" -d @FILENAME DESTINATION`.
+E.g.
+
+```bash
+curl --request POST \
+    --header "Content-Type: application/json" \
+    --data @data.json \
+    "https://oqt.ohsome.org/api/indicator"
+```
+
+In reality one wants to use a tool to URL encode a valid GeoJSON file for requests. In the next section Python is used to archive this.
+
+
+## Request Examples using Python and `requests` library
+
+### Request an Indicator for a pre-definied region
+
+#### GET request
+
+```python
+from urllib.parse import urlencode
+import requests
+
+# Note the question mark (`?`) marking the start of a query string.
+url = "https://oqt.ohsome.org/api/indicator?"
+parameters = {
+    "name": "GhsPopComparisonBuildings",
+    "layerName": "building_count",
+    "dataset": "regions",
+    "featureId": "3",
+}
+# Use `urllib.parse.urlencode` to ensure proper URL encoding:
+response = requests.get(url + urlencode(parameters))
+assert response.status_code == 200
+```
+
+#### POST request
+
+```python
+import requests
+
+# Note the *missing* question mark (`?`).
+url = "https://oqt.ohsome.org/api/indicator"
+parameters = {
+    "name": "GhsPopComparisonBuildings",
+    "layerName": "building_count",
+    "dataset": "regions",
+    "featureId": "3",
+}
+response = requests.post(url, json=parameters)
+assert response.status_code == 200
+```
+
+
+### Request an Indicator for a pre-definied region
+
+#### POST request
+
+Note the using of the `json` library to dump the GeoJSON as string to the parameters.
+URL encoding will be done by the requests library.
```

Comment:
Where do you use the `json` library to dump the GeoJSON as string to the parameters?

If you would:
```suggestion
Note the usage of the `json` library to dump the GeoJSON as string to the parameters.
URL encoding will be done by the requests library.
```

Reply:
The question is still open. I don't think the `json` library is used at all in this document.